src/components/instant-apps-measurement/instant-apps-measurement-tool/instant-apps-measurement-tool.tsx

```diff
@@ -0,0 +1,178 @@
+/** @license
+ * Copyright 2022 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Element, Host, h, Prop, VNode, Watch } from '@stencil/core';
+import { ActiveTool, IMeasureConfiguration } from '../../../interfaces/interfaces';
+
+import { loadModules } from '../../../utils/loadModules';
+const base = 'instant-apps-measurement-tool';
+const CSS = {
+  base,
+  hide: `${base}__hide`,
+};
+@Component({
+  tag: 'instant-apps-measurement-tool',
+  styleUrl: 'instant-apps-measurement-tool.scss',
+  scoped: true,
+})
+export class MeasurementTool {
```

Comment:
Update to InstantAppsMeasurementTool for consistent naming across the components